documentation/content/en/books/arch-handbook/boot/_index.adoc

```diff
@@ -901,21 +897,21 @@ The main task for the loader is to boot the kernel. When the kernel is loaded in
 
 [.programlisting]
 ....
-sys/boot/common/boot.c:
+usr/src/stand/common/boot.c:
     /* Call the exec handler from the loader matching the kernel */
-    module_formats[km->m_loader]->l_exec(km);
+    file_formats[fp->f_loader]->l_exec(fp);
 ....
 
 [[boot-kernel]]
 == Kernel Initialization
 
-Let us take a look at the command that links the kernel. This will help identify the exact location where the loader passes execution to the kernel. This location is the kernel's actual entry point.
+Let us take a look at the command that links the kernel. This will help identify the exact location where the loader passes execution to the kernel. This location is the kernel's actual entry point. This command is now excluded from [.filename]#sys/conf/Makefile.i386#. The content that interests us can be found in [.filename]#/usr/obj/usr/src/i386.i386/sys/GENERIC/# after compiling such as the GENERIC kernel in link:https://xmj.github.io/articles/sysadmin/builds_ccache_memcached.html[METAMODE]
```

Comment:
I'm not sure I'd drag metamode in here - build methods have nothing to do with the boor process IMO. Even if you want metamode (here or elsewhere), I'd link to https://wiki.freebsd.org/MetaMode.

Reply:
You are totally right, I just wanted to show how I came to the link command because it is now excluded from the mentioned file. Are you suggesting that I totally remove the mention of METAMOE?